workshop/01-run-your-first-wasm-on-k3d.md

```diff
@@ -0,0 +1,20 @@
+# Run your first spin application on k3d
+
+The shell script below will create a k3d cluster locally with the Wasm shims installed and containerd configured.
```

Comment:
Can we add or link to resources that explain what the containerd shim is and runwasi?